src/test/java/org/jabref/model/database/event/UpdateTimestampListenerTest.java

```diff
@@ -0,0 +1,99 @@
+package org.jabref.model.database.event;
+
+import org.jabref.gui.BasePanel;
+import org.jabref.logic.preferences.TimestampPreferences;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.preferences.JabRefPreferences;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class UpdateTimestampListenerTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private BibDatabase database;
+
+    @Before
+    public void setUp(){
+        database = new BibDatabase();
+    }
+
+    @Test
+    public void updateTimestampEnabled(){
+        final String TIMESTAMP_FIELD = "timestamp";
```

Comment:
I'm not sure about our conventions, but I think constant variables are still denoted by small letters. @koppor ?